exe/floe

```diff
@@ -7,15 +7,34 @@ require "optimist"
 opts = Optimist.options do
   version "v#{Floe::VERSION}\n"
   opt :workflow, "Path to your workflow json", :type => :string, :required => true
-  opt :inputs, "JSON payload to input to the workflow", :type => :string, :default => '{}'
-  opt :credentials, "JSON payload with credentials", :type => :string, :default => '{}'
+  opt :inputs, "JSON payload to input to the workflow", :type => :string, :default => "{}"
+  opt :credentials, "JSON payload with credentials", :type => :string, :default => "{}"
+  opt :runner, "Type of runner for docker images", :type => :string, :default => "docker"
+  opt :runner_option, "Options to pass to the runner", :type => :string, :multi => true
```

Comment:
A better way to do this might be `:type => :strings`, and leave out the multi true.  In this mode, you specify each option with spaces, so the runner options would look like:

```diff
- --runner-option namespace=default --runner-option server=https://k8s.example.com:6443 --runner-option token=my-token
+ --runner-options namespace=default server=https://k8s.example.com:6443 token=my-token
```

Reply:
Ah yeah that is better